docs/book/driver-deployment/upgrade.md

```diff
@@ -4,14 +4,76 @@
 **Note: To upgrade "vSphere with Tanzu" follow vSphere documentation. Following instructions are applicable to native vanilla Kubernetes Cluster deployed on vSphere.**
 
 Before upgrading driver, please make sure to keep roles and privileges up to date as mentioned in the [roles and privileges requirement](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/prerequisites.html#roles_and_privileges).
-
 Each release of vSphere CSI driver has a different set of RBAC rules and deployment YAML files depending on the version of the vCenter on which Kubernetes Cluster is deployed.
 
+## If you have RWM volumes backed by vSAN file service deployed using vSphere CSI Driver, please refer to the following steps before upgrading vSphere CSI Driver
+
+vSAN file share volumes (Read Write Many vSphere CSI Volumes) become inaccessible after upgrading the CSI driver. Refer to this [issue](https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/1216) for more detail.
+This issue is present in vSphere CSI Driver v2.0.0 to v2.2.1. Fix for this issue will be provided in `v2.3.0` release. New Pods with file share volumes created using v2.3.0 will not run into this issue.
```

Comment:
Fixes are being back ported to older releases as well right. This may have to be revised in future, so better to have the exact list here.

Reply:
@chethanv28 will be taking care of this when he is releasing patches for older releases.